app/scripts/controllers/create.js

```diff
@@ -52,27 +52,43 @@ angular.module('openshiftConsole')
 
         // List image streams and templates in the both the shared `openshift`
         // namespace and the project namespace.
-        DataService.list("imagestreams", {namespace: "openshift"}, function(imageStreams) {
-          $scope.openshiftImageStreams = imageStreams.by("metadata.name");
-        });
+        DataService
+          .list("imagestreams", {namespace: "openshift"}, null, {
+            errorNotification: false
```

Comment:
I'd leave `errorNotification` on. Otherwise you'll need to handle the errors here, but the default error handling should be OK.